src/EventsColumn2.js

```diff
@@ -12,6 +12,66 @@ import {
 import FirebaseConfig from './FirebaseConfig'
 import EventCard from './EventCard'
 
+import * as moment from 'moment'
+import 'moment-recur'
+
+import Sort from './Sort'
+
+let lastMonday = new Date()
+// Ist lastMonday NICHT Montag?
```

Comment:
delete all the comments in this file

Reply:
like to keep it, to understand my code in three weeks aswell ;)